src/setfit/modeling.py

```diff
@@ -29,10 +31,11 @@ def __init__(self, model, max_seq_length: int, add_normalization_layer: bool) ->
 class SetFitModel(PyTorchModelHubMixin):
     """A SetFit model with integration to the Hugging Face Hub."""
 
-    def __init__(self, model_body=None, model_head=None):
+    def __init__(self, model_body=None, model_head=None, multi_target_strategy="one-vs-rest"):
```

Comment:
I think we should set this to `None` so we can keep logistic regression as the default:
```suggestion
    def __init__(self, model_body=None, model_head=None, multi_target_strategy=None):
```